### DISCUSSION TITLE
CM4 Module Installation

### DISCUSSION THREAD

**Discussion by NIDcornifleur** [mcguirepr89/BirdNET-Pi#421] [General]
Hello...
New on BirdNet. I installed on a Rpi 4 and everything works fine. 
Now, I install on a CM4, the sound capture works very well but no result, still in analysis mode.
The picture is not representative... many more birdsongs have been captured with much more strength.. but no results appear. Thanks for your suggestions.
![1](https://user-images.githubusercontent.com/20214731/177056904-d3529516-0ebc-46a7-9f2a-6e88e8f1f680.png)

**Comment by mcguirepr89** [comment]
Hi, @NIDcornifleur -- thanks for your interest in getting BirdNET-Pi running on the CM4. Yours will be the first of which I am aware.

If you don't mind, could you please provide some log output? Whatever you can gather from your "View Log" page, and, if you don't mind sharing your location information and username here, you could also upload your `$HOME/installation-$(date "+%F").txt` log and I would be happy to take a look at potential failings during installation.

My very best regards,
Patrick

**Comment by NIDcornifleur** [comment]
Hello Patrick;

Thank you for your quick response.

Finally, there were a few results after put a better microphone !!
 

You will find the installation as an attachment and here is a part of the view log:

`11:27:38---Files loaded
11:27:39---RECORDING_LENGTH set to 15
11:27:46---BirdNET-Pi/birdnet/bin/python3 BirdNET-Pi/scripts\analyze.py --i BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:30.wav --o BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:30.wav.csv --lat 45.XX --lon -72.XX --week 25 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7
11:27:46---ANALYZING AUDIO... DONE! Time 2.2 SECONDS
11:27:46---WRITING RESULTS TO BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:15.wav.csv ... DONE! WROTE 0 RESULTS.
11:27:47---READING AUDIO DATA... DONE! READ 5 CHUNKS.
11:27:47---Date-time: 2022-07-04 11:27:30
11:27:51---0.0;3.0-('Mniotilta varia_Paruline noir et blanc', 0.055217646)
11:27:51---3.0;6.0-('Scolopax minor_Bécasse d’Amérique', 0.037581652)
11:27:51---6.0;9.0-('Scolopax minor_Bécasse d’Amérique', 0.03987602)
11:27:51---9.0;12.0-('Human_Human', 0.0)
11:27:51---12.0;15.0-('Poecile atricapillus_Mésange à tête noire', 0.087311625)
11:27:54---Files loaded
11:27:54---RECORDING_LENGTH set to 15
11:28:01---BirdNET-Pi/birdnet/bin/python3 BirdNET-Pi/scripts\analyze.py --i BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:45.wav --o BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:45.wav.csv --lat 45.XX --lon -72.XX --week 25 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7
11:28:02---ANALYZING AUDIO... DONE! Time 3.4 SECONDS
11:28:02---WRITING RESULTS TO BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:30.wav.csv ... DONE! WROTE 0 RESULTS.
11:28:02---READING AUDIO DATA... DONE! READ 5 CHUNKS.
11:28:02---Date-time: 2022-07-04 11:27:45
11:28:04---0.0;3.0-('Mniotilta varia_Paruline noir et blanc', 0.13369507)
11:28:04---3.0;6.0-('Piranga olivacea_Piranga écarlate', 0.050176878)
11:28:04---6.0;9.0-('Chordeiles minor_Engoulevent d’Amérique', 0.074911125)
11:28:04---9.0;12.0-('Setophaga pinus_Paruline des pins', 0.04544514)
11:28:04---12.0;15.0-('Chordeiles minor_Engoulevent d’Amérique', 0.116798416)
11:28:08---Files loaded
11:28:08---RECORDING_LENGTH set to 15
11:28:18---BirdNET-Pi/birdnet/bin/python3 BirdNET-Pi/scripts\analyze.py --i BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:28:00.wav --o BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:28:00.wav.csv --lat 45.XX --lon -72.XX --week 25 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7
11:28:18---ANALYZING AUDIO... DONE! Time 2.3 SECONDS
11:28:18---WRITING RESULTS TO BirdSongs/July-2022/04-Monday/2022-07-04-birdnet-11:27:45.wav.csv ... DONE! WROTE 0 RESULTS.
11:28:18---READING AUDIO DATA... DONE! READ 5 CHUNKS.
11:28:18---Date-time: 2022-07-04 11:28:00
11:28:21---0.0;3.0-('Chordeiles minor_Engoulevent d’Amérique', 0.31942984)
11:28:21---3.0;6.0-('Poecile carolinensis_Mésange de Caroline', 0.08475922)
11:28:21---6.0;9.0-('Chordeiles minor_Engoulevent d’Amérique', 0.2648535)
11:28:21---9.0;12.0-('Poecile carolinensis_Mésange de Caroline', 0.26517522)
11:28:21---12.0;15.0-('Chordeiles minor_Engoulevent d’A`

[installation-2022-07-03.txt](https://github.com/mcguirepr89/BirdNET-Pi/files/9040691/installation-2022-07-03.txt)

So everything works with the CM4 and the Pi 4. I will also try with a Zero 2 W. 
Also, I'm going to make a portable one that will go well with the NIDcornifleur [https://nidcornifleur.blogspot.com/2020/12/le-nidcornifleur.html](url)

Regards
Roméo (Location: near Montreal, Canada)
